skbio/sequence/tests/test_interval.py

```diff
@@ -0,0 +1,55 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, scikit-bio development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+from __future__ import absolute_import, division, print_function
+
+import unittest
+
+from skbio.sequence import IntervalMetadata, Interval, Feature
```

Comment:
Yeah.  That's out of date.  All of those tests are being moved over to test_sequence.py.

Thanks for catching that!